### ISSUE TITLE
[Submission Table Options] | Editorial View - Submission panel table, when tabbing through options with 3 dots options the next tab refocuses elsewhere

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12876]
## Issue Description
This imacts keyboard users.  Originally found as a standard accessibility test for each new feature.  The new media files area highlighted this potential issue for useability with keyboard.  It was then discovered that the root cause is any table with 3 dot options that is contained within a panel will lose the focus after tabbing.  You will need to find the spot by tabbing several times after it is reset to get back to the same spot.  When there is no panel (like the Users & Roles table) then the focus works correctly and will continue from where the 3 dot option was choosen.  

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Find any table that has the 3 dots options by using tab
2. Press enter to open up the 3 dot options
3. Press tab again the focus is lost on the 3 dot option and starts at the panel window again.  
4. You can continue presssing tab a few more times to see where you were reset to.  Usually back to the top of the panel.

Expected:
After pressing tab the current location should be rememebered and the next tab should start off where you left off.  Tabbing should follow a linear sequence.  You can see it working correctly in the Users & Roles Settings table with the list of users.  Tab through until you hit one with the 3 dots options, press enter and then tab again and again.  You will start off where you left off.  

Actual:
After pressing tab the focus starts at the top of the panel windows far away from the table with the options.

## What application are you using?
 OJS, OMP or OPS version 3.5, 3.6(main)

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual / Keyboard / Laptop

## Action
// Describe action that can be taken to remediate the issue. 

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

<img width="178" height="151" alt="Image" src="https://github.com/user-attachments/assets/ebcb45fb-d7e7-46be-985d-940334506cd8" />

**Comment by israelcefrin** [comment]
@Tribunal33  The flying menu that opens from the "3 dots" button is dynamically set with `tabindex="0"`. This forces that the next element using the `TAB` key will have `tabindex="1"`. 
A recommended remediation is removing the tabindex parameter since the menu is hidden from keyboard interaction.

<img width="1920" height="819" alt="HTML code showing tabindex set with value equal to zero" src="https://github.com/user-attachments/assets/aad11634-61e5-429e-9944-009c16a78544" />
